buildenv/jenkins/common/variables-functions.groovy

```diff
@@ -637,14 +637,38 @@ def get_date() {
 }
 
 /*
-* Set TESTS_TARGETS, indicating the level of testing
-* and sets TEST_FLAG for all targets if defined in variable file
+* Set TESTS_TARGETS, indicating the level of testing.
 */
 def set_test_targets() {
-    TESTS_TARGETS = params.TESTS_TARGETS
-    if (!TESTS_TARGETS) {
-        // set default TESTS_TARGETS for pipeline job (fetch from variables file)
-        TESTS_TARGETS = get_default_test_targets()
+    TESTS_TARGETS = (params.TESTS_TARGETS) ? params.TESTS_TARGETS.trim() : "none"
```

Comment:
"none" and 'none' be consistent. As there is no no interpolation single quotes will do.

Reply:
done